packages/web/src/components/HomepageFooter/HomepageFooter.tsx

```diff
@@ -1,7 +1,7 @@
 import EmailButton from "./EmailButton";
 
 export const HomepageFooter: React.FC = () => (
-  <footer className="bg-white rounded-lg shadow max-w-2xl  p-4 flex w-full items-center justify-between">
+  <footer className="bg-white rounded-[0.75rem] shadow max-w-2xl  p-2 flex w-full items-center justify-between">
```

Comment:
```suggestion
  <footer className="bg-white rounded-[0.75rem] shadow max-w-2xl  px-2 py-3 flex w-full items-center justify-between">
```

This is the only change i'd make, the rest looks great thank you so much! ❤️